redash/cli/data_sources.py

```diff
@@ -129,7 +127,7 @@ def new(name=None, type=None, options=None, organization="default"):
 
         for k, prop in schema["properties"].items():
             required = k in schema.get("required", [])
-            default_value = "<<DEFAULT_VALUE>>"
+            default_value = prop.get("default") or "<<DEFAULT_VALUE>>"
```

Comment:
In the new version of click, the default value `"<<DEFAULT_VALUE>>"` will fail in type checking.

Reply:
What's the error?

Reply:
For example, in the schema of a data source of "pg," the "port" is of number type. Here, the default value for Click is set as `<<DEFAULT_VALUE>>`. Due to the mismatched data types, the operation fails.

Reply:
Is there any abnormal information clicked here? Please post it if you have one, thanks!
And you can change code to `prop.get("get", "<<DEFAULT_VALUE>>")`

Reply:
No abnormal, but the test case "test_interactive_new" will be blocked.

Reply:
I'm a bit confused about what's going on, but the change seems strictly an improvement over what was there before from what I can tell, so I *think* this should be good to go in. I'd need to play with the command/code a bit to be sure though.

Reply:
```
[N] ❯ ./manage.py ds new
[2023-07-07 09:43:26,593][PID:65737][INFO][xmlschema] Resource 'XMLSchema.xsd' is already loaded
Name: test
Select type:
1. athena
2. graphite
3. mongodb
4. couchbase
5. pg
...
[1-29]: 5
User (optional):
Password (optional):
Host (optional):
Port (optional):
Error: '<<DEFAULT_VALUE>>' is not a valid integer.
Port (optional):
```

This is a detailed example where the new version of "click" verifies whether the default type matches the expected value type.